posthog/hogql/database/schema/test/test_event_sessions.py

```diff
@@ -66,35 +66,35 @@ def test_with_alias_table(self):
             """
                 SELECT e.event
                 FROM events e
-                WHERE e.event = '$pageview'
+                WHERE e.team_id = 1
             """
         )
 
         compare_operators = self._compare_operators(query, "e")
 
         assert len(compare_operators) == 1
         assert compare_operators[0] == ast.CompareOperation(
-            left=ast.Field(chain=["event"]),
+            left=ast.Field(chain=["team_id"]),
```

Comment:
How come you're comparing the `team_id` here? This field is something that gets handled outside of HogQL... so these queries would have it double. It's in many ways reserved in HogQL (e.g. you can't make an alias with this name), so seeing it in tests makes me automatically wonder if something is wrong and if we're leaking data... 🤔

Reply:
Because we not intentionally exclude `event` from the `where` clause, so just wanted to pick any other field for the tests instead. But I get it being possibly weird seeing this in a test